source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperFixture.cs

```diff
@@ -268,6 +288,37 @@ public async Task ExecutionWithEKS_AwsCLIAuthenticator()
             TestScriptInReadOnlyMode(wrapper).AssertSuccess();
         }
 
+        [Test]
+        [WindowsTest] // This test requires the aws cli tools. Currently only configured to install on Windows
+        public async Task ExecutionWithEKS_AwsCLIAuthenticator_WithExecFeatureToggleEnabled()
+        {
+            await InstallTools(InstallAwsCli);
+
+            //set the feature toggle
+            variables.SetStrings(KnownVariables.EnabledFeatureToggles,
+                                 new[]
+                                 {
+                                     FeatureToggle.KubernetesAuthAwsCliWithExecFeatureToggle.ToString()
+                                 },
+                                 ",");
+
+            // Overriding the cluster url with a valid url. This is required to hit the aws eks get-token endpoint.
```

Comment:
Do you mean valid as in 'the format of this url is valid', or valid as in 'you can reach this'?

Reply:
I honestly don't know. I copied the test above this one and tweaked it. The comment came from there.